core/templates/pages/topic-editor-page/modal-templates/create-new-subtopic-modal.controller.ts

```diff
@@ -88,8 +94,12 @@ angular.module('oppia').controller('CreateNewSubtopicModalController', [
       ctrl.errorMsg = null;
     };
     ctrl.isSubtopicValid = function() {
-      return Boolean(ctrl.editableThumbnailFilename &&
-          ctrl.subtopicTitle && ctrl.htmlData);
+      return Boolean(
+        ctrl.editableThumbnailFilename &&
+        ctrl.subtopicTitle &&
+        ctrl.htmlData &&
+        ctrl.editableUrlFragment &&
+        ctrl.checkUrlFragmentValidity());
```

Comment:
The "action" here is too vague. Just do isUrlFragmentValid() -- this implies that it returns a boolean.